dao-user-reference.adoc

```diff
@@ -0,0 +1,192 @@
+
+== BSQ Token
+
+Contributors earn BSQ for working on Bisq, and traders spend BSQ to trade on Bisq. We've covered this basic dynamic in <<user-dao-intro#,this doc>> and https://www.youtube.com/watch?v=pNvOZlIDYEQ&list=PLFH5SztL5cYPAXWFz-IMB4dBZ0MEZEG_e[this video series^].
+
+But what actually _is_ BSQ, and where does it come from?
+
+=== More About BSQ
+
+A single BSQ token is denominated as 100 satoshis (i.e., it's represented on the Bitcoin blockchain as 0.00000100 BTC). This bitcoin is 'colored' with additional properties that causes Bisq software to recognize it as BSQ.
+
+BSQ can only be created in one of two ways:
+
+1. through the genesis distribution (detailed below)
+2. through an issuance transaction (compensation request or reimbursement request)
+
+While BSQ is just bitcoin, it should only be stored in a Bisq wallet.
+
+[sidebar]
+.Warning
+--
+WARNING: **Please do not try to send BSQ to a non-Bisq wallet, or to otherwise create your own BSQ transactions**. The rules to form a valid BSQ transaction are rather complex, and even the slightest error can result in invalid BSQ. Even the <<dao/specification#,technical spec>> does not cover every single detail—the only complete reference is the source code. Also note that future code changes may break custom transactions or implementations. Bisq will NOT provide support for any such cases.
+--
+
+Ultimately, BSQ is stored in a Bitcoin wallet, but the Bisq software takes several measures to keep BSQ and BTC separate to prevent mixing the two by accident. On disk, BTC and BSQ wallets are stored in separate files. In the Bisq interface, BSQ wallets are prefixed with a 'B' to prevent users from mistakenly using BSQ as BTC.
+
+As an additional protection measure, Bisq wallets use a special BIP44 derivation path that other wallets won't understand.
+
+For user convenience, the built-in Bisq wallets for BTC and BSQ share the same seed.
+
+More details are in the <<dao/specification#,technical specification>>.
+
+### BSQ Genesis Distribution
+
+As you may have noticed, the Bisq DAO did not launch with a supply of zero BSQ tokens. So where did the initial supply of BSQ tokens come from?
+
+In short, the initial supply of BSQ tokens was created to account for all the contributions to the Bisq project before the mainnet launch of the Bisq DAO.
+
+These contributions can be divided into 2 segments.
+
+1. _Before October 2017_
++
+In the early days of the project, before Bisq tracked compensation requests and voting results, Bisq collected donations. Over time, about 25 BTC was collected from 144 people. In October 2017, as part of the implementation of the <<dao/phase-zero#the-bsq-token,Phase Zero>> plan, it was decided that 2.5 BTC of the total would be distributed back to these past contributors as BSQ where each person would receive an amount proportional to their prior contributions.
```

Comment:
"about 25 BTC was collected from 144 people." that is wrong. We got donations from anybody (dont know hwo many) it is not related to the  144 contributors from the orig. genesis....

Reply:
I think the 25 BTC  from the donation are not relevant here as number, maybe just mention that we fund the genesis tx from our donations.